docs/quiz/quiz.yaml

```diff
@@ -0,0 +1,65 @@
+paths:
+  /quizzes:
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - Quizzes
+      summary: Creates a new quiz.
+      description: Creates a new quiz with and saves it to the database.
+      produces:
+        - application/json
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#definitions/quizBody'
```

Comment:
Add / after # symbol like this $ref: '#/definitions/quizBody'

Reply:
changed